CHANGELOG/CHANGELOG-2.2.md

```diff
@@ -0,0 +1,61 @@
+# Changelog since v2.1.0
```

Comment:
Can you also add an extra header at the top
```
# Release notes for v2.2.0

[Documentation](https://kubernetes-csi.github.io)

```

Reply:
Sure.